src-built-in/components/windowTitleBar/src/stores/windowTitleBarStore.js

```diff
@@ -45,7 +51,7 @@ var Actions = {
 			windowTitleBarStore.setValue({ field: "hackScrollbar", value: (windowTitleBarConfig.hackScrollbar !== false) });
 
 			// Set by calling WindowClient.setTitle() || from config "foreign.components.Window Manager.title"
-			var title = FSBL.Clients.WindowClient.title || windowTitleBarConfig.title;
+			var title = FSBL.Clients.WindowClient.title || windowTitleBarConfig.title || displayName;
```

Comment:
```suggestion
			const title = FSBL.Clients.WindowClient.title || windowTitleBarConfig.title || displayName;
```

Reply:
A very limited refactoring to some code. This is an old code that might be refactored one day but not at the moment. I'm keeping var as var and const as const and avoiding any rename.